test/crypten/test_hook.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+import crypten
+import torch
+import syft
+
+
+@pytest.fixture(scope="function")
+def model():
+    l_in, l_h, l_out = 32, 16, 2
+    model = crypten.nn.Sequential(
+        [crypten.nn.Linear(l_in, l_h), crypten.nn.ReLU(), crypten.nn.Linear(l_h, l_out)]
+    )
+    return model
+
+
+def test_send_module(workers, model):
+    alice = workers["alice"]
+    model_cmp = model.copy()
```

Comment:
Q: Does it make sense to have a separate test for ```module.copy```?

Reply:
Yeah, I will add a separate test